### DISCUSSION TITLE
那个版本没有通过fid获取群公告的bug

### DISCUSSION THREAD

**Discussion by yujumpjump** [mamoe/mirai#2522] [General]
我想获取群公告。但是最新版本有bug。之前其他版本有获取这个群公告功能吗（没有bug）的版本

**Comment by cssxsh** [comment]
https://github.com/mamoe/mirai/blob/dev/docs/UsingSnapshots.md

如果你是 `mirai-core` 项目可以引用快照

**Reply by yujumpjump** [reply]
我使用的simbot3这个框架来进行开发的 通过maven 导入的依赖 我是否可以通过maven 导入特定的mirai-core版本

**Reply by cssxsh** [reply]
> 我使用的simbot3这个框架来进行开发的 通过maven 导入的依赖 我是否可以通过maven 导入特定的mirai-core版本

我给的链接你没看吗

**Reply by yujumpjump** [reply]
看了 不好意思 可能是我没有看明白 是不是只有使用mirai-core 项目 才可以使用这个快照吗

**Reply by cssxsh** [reply]
simbot3 基于 mirai-core
你的项目基于 simbot3 

那你说你的项目有没有基于 mirai-core

pom 里没有对应配置那就自己加上去啊

**Reply by yujumpjump** [reply]
我已经解决这个问题了 麻烦大佬了 我没有在通过fid 获取文件 而且通过索引值获取的

**Comment by StageGuard** [comment]
我简单看了一下，你用的 simbot 框架在文档里明确说明不会支持 mirai dev 版本，所以等待等 mirai 发布下一个 release 版本
https://github.com/simple-robot/simbot-component-mirai#mirai%E4%BE%9D%E8%B5%96%E8%AF%B4%E6%98%8E

**Reply by yujumpjump** [reply]
好的 谢谢
